listenbrainz/webserver/static/js/src/RecentListens.tsx

```diff
@@ -566,6 +578,13 @@ export default class RecentListens extends React.Component<
       currentUser,
     } = this.props;
 
+    const isNewestButtonDisabled = listens?.[0]?.listened_at >= latestListenTs;
+    const isNewerButtonDisabled =
+      !previousListenTs || previousListenTs >= latestListenTs;
+    const isOlderButtonDisabled =
+      !nextListenTs || nextListenTs <= oldestListenTs;
+    const isOldestButtonDisabled =
+      listens?.[listens?.length - 1]?.listened_at <= oldestListenTs;
```

Comment:
if listens is null, will `listens?.length - 1` return a number or some other placeholder? or will the first `listens?.` ensure that this doesn't happen?

Reply:
If listens is null `listens?` will evaluate as `undefined` (same for the other `?.` operators in that line)